src/main/scala/info/bethard/timenorm/formal/Types.scala

```diff
@@ -494,10 +494,16 @@ case class AfterRI(interval: Interval,
   * @param startInterval first interval
   * @param endInterval   second interval
   */
-case class Between(startInterval: Interval, endInterval: Interval) extends Interval {
+case class Between(startInterval: Interval, endInterval: Interval, startIncluded: Boolean, endIncluded: Boolean) extends Interval {
```

Comment:
Make these `startIncluded: Boolean = false` and `endIncluded: Boolean = false`. Right now, the tests are failing because these parameters have no defaults.